audio/out/ao_wasapi.h

```diff
@@ -47,7 +47,7 @@ HRESULT wasapi_change_init(struct ao* ao, bool is_hotplug);
 void wasapi_change_uninit(struct ao* ao);
 
 enum wasapi_thread_state {
-    WASAPI_THREAD_FEED = 0,
+    WASAPI_THREAD_FEED = 1,
```

Comment:
Why?

Reply:
The client thread is waken up with `thread_wakeup` after it's created and since `wasapi_state` is zeroed the thread will never run without this change.

Reply:
If initial state (zeroed) is invalid, it should be exlicitly initialized to value.